momaland/envs/breakthrough/breakthrough_check.py

```diff
@@ -0,0 +1,43 @@
+"""MO Breakthrough check."""
```

Comment:
Might put this in the main of the env itself ?

Reply:
There's a discussion on whether other environments should have this and where, so I'm going to resolve this one